includes/definitions/zxr10.yaml

```diff
@@ -3,6 +3,11 @@ group: zxr10
 text: 'ZTE ZXR10'
```

Comment:
OH, here is the problem, group is zxr10 instead of zte.  There are not any other devices in the zxr10 group and no

Looking closer, the only real group is zyxel... which just adds fallback to ZYXEL-ES-COMMON for os information.

Just delete group here and leave mib_dir to zte.

You can set group to zyxel if it adds hardware/version/serial number.

Reply:
zyxel - is a vendor
zte - is an another vendor. is's not the same to zyxel

zxr10 is zte's product line of L3 switches

in librenms there is another product line from zte called zxdsl (ADSL modems/routers I think)

So I think it needs to just change zxr10 group to zte

Reply:
Oops, too early in the morning.  I agree with this fix :)